.github/workflows/protoc.yml

```diff
@@ -0,0 +1,33 @@
+
+name: Auto-generate gRPC files
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+
+    protoc:
+        name: Create gRPC autogenerated files and commit
```

Comment:
Why not do this as part of the PR rather than in main? You would have GH actions commit again to the repo. I'd go with the pyansys-ci-bot in fact. We have some references on how to do this. I can take care of it if you want.

Reply:
still working on getting the protoc setup automatically. but sure, you are the expert on this stuff, so i'm happy to follow your guidance here. i'll let you know once my action should work and then you can make the suggested change, if that is fine for you. thanks!

Reply:
Sounds good! Far from being an expert though 😄

Reply:
the most expirenced person with all the github stuff i know at least =)

Reply:
ok, it finally seems to work. my idea was to check if the proto creation works fine in the ci_cd.yml (so on pr creation with the "protoc" job) and then actually create and commit the grpc definition files on merge of the pr to main, with my protoc.yml workflow. making sure they don't get affected by the gitignore entry by using "git add -f pkg/allieflowkitgrpc/". feel free to change this however you see fit.